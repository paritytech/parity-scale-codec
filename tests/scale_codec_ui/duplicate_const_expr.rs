#[derive(::parity_scale_codec::Encode)]
#[codec(crate = ::parity_scale_codec)]
pub enum Enum {
    #[codec(index = MY_CONST_INDEX)]
    Variant1,
    #[codec(index = MY_CONST_INDEX)]
    Variant2,
}

const MY_CONST_INDEX: u8 = 1;

fn main() {}
