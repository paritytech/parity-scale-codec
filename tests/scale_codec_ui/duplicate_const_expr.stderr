error: index value `MY_CONST_INDEX` is assigned more than once
 --> tests/scale_codec_ui/duplicate_const_expr.rs:6:21
  |
6 |     #[codec(index = MY_CONST_INDEX)]
  |                     ^^^^^^^^^^^^^^
